tests/security/tpm2/tpm2_engine/tpm2_engine_ecdsa_operation.pm

```diff
@@ -28,7 +29,8 @@ sub run {
     assert_script_run "cd $test_dir";
     assert_script_run "echo tpm2test > $test_file";
     assert_script_run "tpm2tss-genkey -a ecdsa -s 2048 $my_key";
-    assert_script_run "openssl ec -engine tpm2tss -inform engine -in $my_key -pubout -outform pem -out $my_key.pub";
+    my $extra_opts = (is_sle('<=15-SP5') && !is_tumbleweed) ? "-engine tpm2tss -inform engine" : "-provider tpm2";
```

Comment:
from `CONTRIBUTING.md`:

```
Avoid is_tumbleweed(): Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check. 
```

if we follow this approach, the tumbleweed new options should be the default, and we should use older options for `<=15-SP6` products

Reply:
thanks, let me fix that